main/sceneManager.h

```diff
@@ -16,29 +16,24 @@
 class SceneManager
 {
 public:
+    SceneManager();
     bool loadModel(std::string_view objPath);
 
     std::vector<objects::EnergyCollector *> getEnergyCollectors();
-
-    std::vector<objects::TriangleObj *> getModelTriangles()
-    {
-        std::vector<objects::TriangleObj *> v;
-        v.reserve(model_triangles_.size());
-        for (const auto &t : model_triangles_)
-        {
-            v.push_back(t.get());
-        }
-        return v;
-    }
-
+    std::vector<objects::TriangleObj *> getModelTriangles();
     std::vector<objects::TriangleObj *> getReferencePlate();
     const objects::SphereWall &getSphereWall();
+    const generators::PointSource &getPointSource();
 
 private:
     // Disallow copy and assign.
     SceneManager(const SceneManager &) = delete;
     SceneManager &operator=(const SceneManager &) = delete;
-    std::vector<std::unique_ptr<objects::TriangleObj>> model_triangles_;
+    std::vector<std::unique_ptr<objects::TriangleObj>> _modelTriangles;
+    std::vector<std::unique_ptr<objects::EnergyCollector>> _energyCollectors;
+
+    //  METHODS
+    void createCollectors();
```

Comment:
Fix the declaration order:

https://google.github.io/styleguide/cppguide.html#Declaration_Order

Reply:
![image](https://user-images.githubusercontent.com/45213495/96495121-aa0eb280-1247-11eb-8142-ca25dbd1729b.png)
do you mean somethinkg like this or adding inline keyword? ;)

Reply:
How does inline keyboard fixes the declaration order?

What you have is close, but not quite: order is this (copied from the docs, please read carefully next time):

> types (including typedef, using, and nested structs and classes), constants, factory functions, constructors and assignment operators, destructor, all other methods, data members.

Constructors first, then methods, data members are at the end.

Reply:
ok, fixed, thanks 👍